quaint/src/connector/postgres/native/mod.rs

```diff
@@ -292,6 +296,21 @@ impl PostgreSql {
         })
     }
 
+    /// Create a new websocket connection to managed database
+    pub async fn new_with_websocket(url: PostgresWebSocketUrl) -> crate::Result<Self> {
+        let client = connect_via_websocket(url).await?;
+
+        Ok(Self {
+            client: PostgresClient(client),
+            socket_timeout: None,
+            pg_bouncer: false,
```

Comment:
Should `pg_bouncer` be always set to false, or should it be dynamically set depending on the query parameters in the connection URL?

Reply:
DBs that will be connecting to are guaranteed to not have pgbouncer in front of them